.github/workflows/concurro.yml

```diff
@@ -0,0 +1,21 @@
+name: Concurro CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build:
+    name: "Build and test"
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout for build
+      uses: actions/checkout@v4
+    - name: Set up JDK
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin'
+        java-version: 21
+    - name: Build with Maven
+      run: mvn -V -ntp -P staging verify
```

Comment:
![image](https://github.com/eclipse-ee4j/glassfish-concurro/assets/11896137/b2edc1f4-4876-48db-a10b-329e65102c67)
Seems that with 
- #78

the `staging` profile needs to be activated.